### PR TITLE
BUG make sure to not munge prompt on zsh

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

File: recipe/activate-gcc.sh [hunk 1]
```diff
@@ -185,4 +187,21 @@ else
     diff -U 0 -rN /tmp/old-env-$$.txt /tmp/new-env-$$.txt | tail -n +4 | grep "^-.*\|^+.*" | grep -v "CONDA_BACKUP_" | sort
     rm -f /tmp/old-env-$$.txt /tmp/new-env-$$.txt || true
   fi
+
+  # fix prompt for zsh
+  if [[ -n "${ZSH_NAME}" ]]; then
```
Comment (isuruf) [inline recipe/activate-gcc.sh:192]:
```suggestion
  if [[ -n "${ZSH_NAME:-}" ]]; then
```

File: recipe/activate-gcc.sh [hunk 2]
```diff
@@ -185,4 +187,21 @@ else
...
+    _conda_clang_precmd() {
+      HOST="${CONDA_BACKUP_HOST}"
+    }
+
+    [[ -z \$precmd_functions ]] && precmd_functions=()
```
Comment (isuruf) [inline recipe/activate-gcc.sh:197]: Why is there a `\` at the beginning?

File: recipe/activate-gcc.sh (hunk 1 above)
Comment (isuruf) [reply recipe/activate-gcc.sh:192]: Can you make the same change in clang recipe? Need this for shell sessions with `-u` set.

File: recipe/activate-gcc.sh (hunk 2 above)
Comment (beckermr) [reply recipe/activate-gcc.sh:197]: I don't know. I parts of this from a stack overflow post.

File: recipe/activate-gcc.sh (hunk 1 above)
Comment (beckermr) [reply recipe/activate-gcc.sh:192]: Yep. Will fix!

Comment (beckermr) [reply recipe/activate-gcc.sh]: done!

File: recipe/activate-gcc.sh (hunk 2 above)
Comment (beckermr) [reply recipe/activate-gcc.sh]: fixed!

**Comment by beckermr** [comment]
We should wait on this until the clang PR has been debugged.

**Comment by beckermr** [comment]
@isuruf the clang version of this is working great. This or can be merged.

**Comment by beckermr** [comment]
bump here @isuruf - I think we should merge this one.

**Comment by sdvillal** [comment]
another bump, this fix works great already in the clang activation feedstock...

**Comment by beckermr** [comment]
I am waiting on code review from @isuruf but am willing to merge in another week if we don't hear back.

**Comment by beckermr** [comment]
thanks @isuruf!

**Pull request merged**